pyccl/halos/profiles_3pt.py

```diff
@@ -0,0 +1,68 @@
+from .profiles import HaloProfile
+
+
+class Profile3pt(object):
```

Comment:
So, although it is great to have a proper implementation of this, the annoying thing is that this is a very, very specific implementation, which does not correspond to a general 3-point function, but to the specific case <u(ku) v1(kv) v2(kv)>. For this reason I'm tempted to say that, for now, the 3,1 part of the connected trispectrum should just involve products of profiles rather than a general 3-point cumulant...

Reply:
hmm, should I change it already?

Reply:
@damonge thoughts on this?

Reply:
Done.`I_1_3` computes now `I_1_1(k) * I_1_2(k')`